docs/DeveloperGuide.md

```diff
@@ -117,28 +117,25 @@ The `UI` component,
 
 **API** : [`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
 
-At high level, Logic Component is responsible for making sense of the user inputs, and modify storage and ui accordingly.
-It is like a controller that glue the other components together.
+At a high level, the `Logic` component is responsible for making sense of the user inputs, and modifying storage and ui accordingly.
+It acts like a controller to glue the other components together.
 
-To be more specific, it interacts with ui component by taking the user input from it, and setting the ui display accordingly.
-It also calls APIs (addPerson, deleteLesson for example) from model component to modify the data representation, and call APIs
-from storage component to save the data to local storage each time the data is modified.
+It interacts with the `UI` component by taking the user input from it, and setting the UI display accordingly.
+It also calls APIs (addPerson, deleteLesson for example) from the `Model` component to modify the data representation, and call APIs
+from the `Storage` component to save the data to local storage each time the data is modified.
 
 The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
 <puml src="diagrams/DeleteSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `delete 1` Command" />
 
-Inside the `Logic` component, there are 4 main components: LogicManager, AddressBookParser, ScheduleParser and Command.
+Inside the `Logic` component, there are 3 main components: `LogicManager`, `AddressBookParser` and `Command`.
```

Comment:
A bit weird writing "AddressBookParser" here, how about simply says "Parser".

Reply:
or says there are 4 main components, as addressBookParser does not implement parser

Reply:
For this section, I interpret it as talking about the various important classes of the Logic component so like the 3 important classes LogicManager, AddressBookParser and the Command classes. Not too sure how we want to phrase it.

Reply:
sure